data/preset/encyclopedia.yaml

```diff
@@ -48,7 +48,8 @@ defines:
                       arrangement:
                         shortdef: 'Arrangement.List(hexpand: true)'
```

Comment:
I think we should also set the arrangement list homogeneous property to false, otherwise all the results will have the  width of the widest one